scripts/pricefeeds/pricefeeds.py

```diff
@@ -299,12 +305,42 @@ def derive_prices(feed):
             print("[Warning] No market route found for %s. Skipping price" % asset)
             continue
 
-        # price convertion to "price for one BTS" i.e.  base=*, quote=core_symbol
-        price_result[asset] = {"mean"    : price_mean,
+        # price conversion to "price for one BTS" i.e.  base=*, quote=core_symbol
+        price_result[asset]['BTS'] = {"mean"    : price_mean,
```

Comment:
Above, you derive the price for `quote` (i.e. short backing asset) and now you store the price in the `BTS` struct? That doesn't feel right.